cmd/prometheus-config-reloader/main.go

```diff
@@ -91,6 +97,12 @@ func main() {
 
 	versionutil.RegisterIntoKingpinFlags(app)
 
+	err := web.Validate(*webConfig)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Unable to validate web configuration file. err: "+err.Error())
```

Comment:
Could we use `level.Error(logger....` instead?

Reply:
If `webConfig` is configured incorrectly, http.Server will fail to start. Similar to `app.Parse` and `version verification`, `os.Stderr` output is more appropriate。

```
	err := web.Validate(*webConfig)
	if err != nil {
		fmt.Fprintln(os.Stderr, "Unable to validate web configuration file. err: "+err.Error())
		os.Exit(2)
	}

	if _, err := app.Parse(os.Args[1:]); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(2)
	}

	if versionutil.ShouldPrintVersion() {
		versionutil.Print(os.Stdout, "prometheus-config-reloader")
		os.Exit(0)
	}

	logger, err := logging.NewLogger(*logLevel, *logFormat)
	if err != nil {
		stdlog.Fatal(err)
	}
```